docs/getting_started/requirements.md

```diff
@@ -34,6 +34,12 @@ These setups are tested by QA and are generally recommended setups. For security
 |Package manager|Composer (recent stable version)|
 |Asset manager|`Node.js` 10.15.3 LTS</br>`yarn` 1.15.2 or higher|
 
+!!! caution "Java version"
+
+    If your Java version is higher than 8, you need to downgrade it to Oracle Java/Open JDK: 8. 
+    You can do it by following your favorite tutorial, for example:
+    [How to downgrade Java on Ubuntu/Kali Linux.](https://flicsdb.com/how-to-downgrade-java-on-ubuntu-kali-linux/)
```

Comment:
Is there any official documentation for debian/red hat/oracle about installing concrete java version?

Reply:
I could find only this: https://developers.redhat.com/blog/2018/12/10/install-java-rhel8/ or we can skip a link altogether.

Reply:
Maybe it is just me, but can't we find a better spot/less confusing way to mention it? Right now it just "caution" under the requirements table - if I did not pay attention reading the whole doc I could get a wrong idea that Java is required for Platform to work and I have to install it in version 8, which is not true. 
But as I said, it may be just me...

Reply:
It is the only place we mention Java in the documentation, but if it's confusing in this format, we can move it inside the table as a plain text under `Oracle Java/Open JDK: 8` requirement.
The second note is in 'Download and configure solr' section.